src/js/edu-benefits/containers/EduBenefitsApp.jsx

```diff
@@ -14,13 +14,13 @@ import PerfPanel from '../components/debug/PerfPanel';
 import RoutesDropdown from '../components/debug/RoutesDropdown';
 
 import { isValidSection } from '../utils/validations';
-import { ensureFieldsInitialized, updateCompletedStatus } from '../actions/index';
+import { ensureSectionInitialized, updateCompletedStatus } from '../actions/index';
```

Comment:
In this case section == page. I was going to create an issue to go back and change it everywhere at once, since I didn't want to make this PR bigger than it already is.